azure-pipelines.yml

```diff
@@ -99,16 +105,13 @@ jobs:
       test_extras: 'dev'
       test_command: 'pytest -p no:warnings --doctest-rst -m "not figure" --pyargs sunpy'
       submodules: false
-
       targets:
-        - wheels_cp3[789]-manylinux*x86_64
-        - wheels_cp3[789]-macosx*
+        - wheels_cp3[78]-manylinux*x86_64
+        - wheels_cp3[78]-macosx*
         - sdist
       dependsOn:
         - py37_test
         - py38_test
         - py39_test
         - py38_test_online
-        - py38_test_32bit
```

Comment:
hmm, perhaps we should change this to allow these to stay? So we run the full lot before a release?

Reply:
I agree but I am unclear how I would logic this at the moment.